### PR TITLE
Fix for licenses problems

### PR DESCRIPTION
**Pull request by dianacle**
Fix problems found with check-binary-licence 
Master Issue: #2592

### CODE REVIEW

**Comment by eolivelli** [comment]
Great work @dianacle 

Can you please add my PR  in order to trigger the validation?
https://github.com/apache/bookkeeper/pull/2600

**Review by dlg99 — APPROVED** [review]
Thank you for taking care of this!

**Comment by dianacle** [comment]
> Great work @dianacle
> 
> Can you please add my PR in order to trigger the validation?
> #2600

**Comment by eolivelli** [comment]
rerun failed tests

**Comment by eolivelli** [comment]
It looks like my script is not finding the binary tarballs.

**Comment by eolivelli** [comment]
In my pr the validation found the files
https://github.com/apache/bookkeeper/pull/2600

File: dev/check-all-licenses [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# LICENSE file. It checked that all jar files mentioned in NOTICE and
+# LICENSE are actually bundled.
+
+# all error fatal
+set -e -x
+
+HERE=$(dirname $0)
+BOOKKEEPER_DIST=$HERE/../bookkeeper-dist
+$HERE/check-binary-license $BOOKKEEPER_DIST/server/target/bookkeeper-server-*-bin.tar.gz
+$HERE/check-binary-license $BOOKKEEPER_DIST/server/all/bookkeeper-all-*-bin.tar.gz
```
Comment (eolivelli) [inline dev/check-all-licenses:31]: The script is wrong!
In my branch it failed during the validation of 'server' package.
But now the validation is passing :)
And now the path of the second and the third files are wrong 

Can you please fix it @dianacle ?

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: dev/check-all-licenses (hunk 1 above)
Comment (dianacle) [reply dev/check-all-licenses]: Done

**Review by eolivelli — APPROVED** [review]
Cool.
Thanks.

I will merge tomorrow if no one else does it

**Comment by eolivelli** [comment]
Committed to master branch,
thank you @dianacle 
now we are ready for a release

**Pull request merged**